communicator/ssh/communicator.go

```diff
@@ -236,7 +298,7 @@ func (c *comm) Download(path string, output io.Writer) error {
 		return nil
 	}
 
-	return c.scpSession("scp -vf "+strconv.Quote(path), scpFunc)
+	return c.scpSession("scp -vf "+path, scpFunc)
```

Comment:
Spaces i think no, but in my case i can use * and download all files, not just one, o use wildcards (internally this is supported by scp). So i don't wan quote paths. If user need - he/she can add quotes for needed paths.